### ISSUE TITLE
Slight Feature Removal. (Loading Amiibo's hotkey)

### ISSUE THREAD

**Issue by G-Spawn** [yuzu-emu/yuzu#5144]
It does not bother me any but I've noticed the 'load amiibo' F2 hotkey that doesn't quite work is still listed under hotkeys. If it's a placeholder that's fine but it could get some user's excited about being able to load amiibo's to find it doesn't quite work as expected.

![image](https://user-images.githubusercontent.com/23653025/101267998-a74f1a80-372c-11eb-98ad-c9dce73f01cd.png)

It's such a slight thing so feel free to close if it's not something worth approaching. I just figured it would save people asking if it was just outright removed for now. Thank you for your time.

**Comment by choppymaster** [comment]
There is also a `load amiibo` option on the first tab

**Comment by G-Spawn** [comment]
> 
> 
> There is also a `load amiibo` option on the first tab

Not that I've seen in any of the version I've used in the last few weeks-months at least. The f2 hotkey seems to be the only means to call it. It's more than likely just the game / the one's I have though so its whatever. 

I've discussed it a little further in the discord but I only have so many of these things to test. If it's a consistent weirdness for others perhaps it could be worth looking into but it's likely just me.
